docs/src/design/decoder/constraints.md

```diff
@@ -422,18 +422,18 @@ Inside a *span* block, group count can either stay the same or decrease by one:
 sp \cdot \Delta gc \cdot (\Delta gc - 1) = 0 \text{ | degree} = 3
 $$
 
-When group count is decremented inside a *span* block, either $h_0$ must be $0$ (we consumed all operations in a group) or we must be executing `PUSH` operation:
+When group count is decremented inside a *span* block, either $h_0$ must be $0$ (we consumed all operations in a group) or we must be executing an operation with an immediate value:
 
 > $$
-sp \cdot \Delta gc \cdot (1 - f_{push})\cdot h_0 = 0 \text{ | degree} = 7
+sp \cdot \Delta gc \cdot (1 - f_{imm})\cdot h_0 = 0 \text{ | degree} = 8
 $$
```

Comment:
If we change the `PUSH` opcode as described in https://github.com/0xPolygonMiden/miden-vm/pull/1496#discussion_r1760576863, the degree here would remain the same.

Reply:
As commented above, I believe this remains the same, since `f_imm` is still degree 5